app/views/redmine_required_field/_settings.html.erb

```diff
@@ -3,28 +3,29 @@
   rrf_all_trackers = RedmineRequiredField.related_tracker_options
   rrf_all_issue_statuses = RedmineRequiredField.related_issue_status_options
   rrf_all_projects = RedmineRequiredField.related_project_options
+  rrf_all_roles = RedmineRequiredField.related_roles
   rrf_issue_required = RedmineRequiredField.settings_issue_required
+  current_roles = RedmineRequiredField.current_roles
```

Comment:
would be good to call this rrf_current_roles, in the same page there can be other partials (coming from other plugins etc.) prefixing the variable helps preventing using same name for different variables

Reply:
you are right. 
but since i realized i'm not unsing it anymore, i can get rid of this variable
coming with next commit